src/crimson/os/seastore/segment_manager/zns.cc

```diff
@@ -137,7 +137,7 @@ static seastar::future<> reset_device(
     ZoneReport(nr_zones),
     [&, nr_zones](auto &range, auto &zr) {
       range.sector = 0;
-      range.nr_sectors = zone_size_sects * nr_zones;
+      range.nr_sectors = (uint64_t)zone_size_sects * nr_zones;
```

Comment:
Not sure why the type of parameter is `uint32_t`.

Probably better to fix the method instead?
```c++
static seastar::future<> reset_device(
  seastar::file &device,
  uint64_t zone_size_sects,
  uint64_t nr_zones)
```

Reply:
Ok, Done.